doc.js

```diff
@@ -378,15 +378,15 @@ define('doc', ['event'], function(event) {
 				return !this.isPresent();
 			},
 
-			'on' : function(eventsName, command, named) {
+			'on' : function(eventsName, command, named, passive) {
```

Comment:
If [this suggestion](https://github.com/elo7/events-amd/pull/5#discussion_r263796641) is accepted, it would be nice to adjust arguments here accordingly.